app/views/shared/_navbar.html.erb

```diff
@@ -42,7 +42,7 @@
     <nav class="collapse navbar-collapse">
       <ul class="navbar-nav ml-auto">
         <li class="nav-item <%= active_class(courses_path) %>">
-          <%= link_to 'Curriculum', courses_path(ref: 'homenav'), class: 'nav-link' %>
+          <%= link_to 'Curriculum', track_path(1, ref: 'homenav'), class: 'nav-link' %>
```

Comment:
What do you think of removing the hardcoded the track id and using `Track.default` instead?

```ruby
# app/models/track.rb

def self.default
  first
end
```

We can even go a step further and create a boolean column in the `tracks` table -  `default`. This gives us the flexibility to update the default course without changing the code.

WDYT?